bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -1238,7 +1238,8 @@
 		<description>Current Power</description>
 		<category>Energy</category>
 		<tags>
-			<tag>sensors</tag>
+			<tag>Measurement</tag>
+			<tag>Power</tag>
```

Comment:
I am not sure if Power or Energy must be used here.

Reply:
I think this binding has some modeling issues because it seems that this power type is used for both Power and Pressure:

https://github.com/openhab/openhab-addons/blob/8b91e2172f48025efab0fa3f7701347e79b71bea/bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml#L744-L751


https://github.com/openhab/openhab-addons/blob/8b91e2172f48025efab0fa3f7701347e79b71bea/bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml#L769-L776

So until that is resolved it might be better to remove the tags?

Reply:
Yes, I agree. I will remove it,

Reply:
Sorry I thought it was already removed. 😆